App/views.py

```diff
@@ -0,0 +1,55 @@
+from django.http import HttpResponse
+from django.shortcuts import render
+import json
+import requests
+import toolforge
+
+def main2(request):
+    if(request.method=='POST'):
+        usr = request.POST['username']
+        SqlQuery1 = "SELECT user_editcount FROM user WHERE user_name='" + usr + "';"
+        SqlQuery2 = "SELECT ss_total_edits FROM site_stats"
+        SqlQuery3 = "select rank from (select user_name, @curRank := @curRank + 1 as rank from user p, ( select @curRank := 0 ) q order by user_editcount DESC) q where user_name='"+usr+"';"
```

Comment:
This does work but it's a bit overcomplicated. You just need to count how many users have more edits.

Reply:
More problematically, the first and third queries should escape the user. Right now you could create a user called something like `'; DROP TABLE user; --` and [things would end badly](https://xkcd.com/327/).

Reply:
I have improvised the rank query.
Also, I have made the input string 'sql injection proof' :)